invenio_app_ils/ui/src/invenio_app_ils/common/api/eitems/eitem.js

```diff
@@ -15,6 +15,18 @@ const del = async eitemPid => {
   return response;
 };
 
+const create = async data => {
+  const resp = await http.post(`${eitemURL}`, data);
+  resp.data = serializer.fromJSON(resp.data);
+  return resp;
+};
+
+const update = async (eitemPid, data) => {
+  const response = await http.put(`${eitemURL}${eitemPid}`, data);
```

Comment:
put is going to replace what's already there as a resource, just to verify that this is the desired behaviour

Reply:
Yes it is the desired behaviour!